config/README.md

```diff
@@ -0,0 +1,112 @@
+# Introduction
+
+This document describes the available options in the JSON config for `status-go`.
+
+The structure of the JSON config is defined in the [`params/config.go`](/params/config.go) file, which also contains details comments on meaning of each option.
+If the descriptions of any options are too vague feel free to [open an issue](https://github.com/status-im/status-go/issues/new).
+
+Example config files can be viewed in the [`config/cli`](/config/cli) folder.
+
+# Important Sections
+
+The JSON config is separated into several sections.
+
+## Node Config
+
+The root of the JSON configuration.
+
+An example of most important settings would include:
+```json
+{
+    "NetworkID": 1,
+    "DataDir": "/tmp/status-go-data",
+    "NodeKey": "123qwe123qwe123qwe123",
+    "Rendezvous": true,
+    "NoDiscovery": false,
+    "ListenAddr": "0.0.0.0:30303",
+    "AdvertiseAddr": "12.34.56.78",
+    "RegisterTopics": ["whispermail"]
+}
+```
+
+If you'd want to enable JSON RPC port you'd need:
+```json
+{
+    "HTTPEnabled": true,
+    "HTTPHost": "0.0.0.0",
+    "HTTPPort": 8545,
+    "APIModules": "eth,net,web3,admin"
+}
+```
+
+In order to adjust logging settings you'd need:
+```json
+{
+    "LogFile": "/var/log/status-go.log",
+    "LogLevel": "INFO",
+    "LogMaxSize": 200,
+    "LogMaxBackups": 5,
+    "LogCompressRotated": true
+}
+```
+Valid `LogLevel` settings are: `ERROR`, `WARN`, `INFO`, `DEBUG`, `TRACE`
+
+## `LightEthConfig`
+
+This config manages [LES](https://github.com/ethereum/wiki/wiki/Light-client-protocol)-realted settings.
+A basic config would include:
+```json
+{
+    "LightEthConfig": {
+        "Enabled": true,
+        "DatabaseCache": 32,
+        "TrustedNodes": [
+            "enode://123qwe123@12.34.56.78:30303",
+            "enode://123qwe123@87.65.43.21:30303"
+        ]
+    }
+}
+```
+The `TrustedNodes` is a good way to get your LES node bootstraped with peers.
+
+## `WhisperConfig` 
+
+If you want your node to relay Whisper(SHH) procotol messages you'll want to include this:
+```json
+{
+    "WhisperConfig": {
+        "Enabled": true,
+        "EnableMailServer": true,
+        "LightClient": false,
+        "MailServerPassword": "status-offline-inbox"
+    }
+}
+```
+The `MailServerPassword` is used for symmetric encryption of history requests.
```

Comment:
we can mention that this password will be used when mail server is added in status app.
@cammellos maybe you have a link to documentation that explains that?

Reply:
I would add that `status-offline-inbox` is the password used by Status mail servers listed on https://fleets.status.im.

Reply:
Good idea.